dm/Dockerfile

```diff
@@ -1,5 +1,4 @@
 FROM golang:1.19.1-alpine3.15 as builder
```

Comment:
do we need to update this image? not sure if the building stage should also be put in safer image

Reply:
If runtime use glibc, builder use glibc

Reply:
here we just parallel produce another docker image, directly change may be dangerous